Sources/DiscoverKit/View/RecommendationRow.swift

```diff
@@ -28,6 +28,7 @@ struct RecommendationRow: View {
                 .font(.headline)
             Text(recommendation.excerpt)
                 .font(.body)
+                .lineLimit(5)
```

Comment:
I think this file contains a bunch of constants, and it might be time to group them into a `private enum`, what do you think?

Reply:
yes! sounds good to me!